pkg/trace/obfuscate/obfuscate.go

```diff
@@ -102,7 +102,7 @@ func (o *Obfuscator) Obfuscate(span *pb.Span) {
 func (o *Obfuscator) ObfuscateStatsGroup(b *pb.ClientGroupedStats) {
 	switch b.Type {
 	case "sql", "cassandra":
-		oq, err := o.ObfuscateSQLString(b.Resource)
+		oq, err := o.ObfuscateSQLString(b.Resource, config.SQLObfuscationConfig{})
```

Comment:
It would be good to maintain the APM feature, only for SQL obfuscation in spans: `features.Has("quantize_sql_tables")` 

If we remove it entirely then there's no way to enable this for APM anymore.

Reply:
Updated: https://github.com/DataDog/datadog-agent/pull/8556/commits/674ec1d578cb119502eff9d4194a8af7b27158ee